composer.json

```diff
@@ -17,21 +17,22 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3",
-        "sonata-project/admin-bundle": "~2.4@dev",
-        "sonata-project/block-bundle": "~2.3@dev",
-        "sonata-project/doctrine-extensions" : "~1.0",
-        "sonata-project/cache-bundle": "~2.1,>=2.1.7",
-        "sonata-project/core-bundle": "~2.3,>=2.3.1",
-        "sonata-project/notification-bundle": "~2.2",
-        "sonata-project/datagrid-bundle": "~2.2@dev"
+        "php": "^5.3 || ^7.0",
+        "symfony/symfony": "^2.3 || ^3.0",
+        "sonata-project/admin-bundle": "^3.0",
+        "sonata-project/block-bundle": "^3.0",
+        "sonata-project/doctrine-extensions" : "^1.0",
+        "sonata-project/cache-bundle": "^2.1.7",
```

Comment:
Nope, cache-bundle will be a 2.x for now.

https://packagist.org/packages/sonata-project/cache-bundle